pkg/types/types.go

```diff
@@ -21,17 +21,22 @@ const (
 	IPv4Width  Width = 4
 	DPortWidth Width = 2
 	ProtoWidth Width = 1
+
+	TimestampWidth Width = 8
 )
 
 const (
 	KeyWidthIPv6 = 2*IPv6Width + DPortWidth + ProtoWidth
```

Comment:
This comment is at its core opinionated. And due to the fact that I have a mild degree of obsession about reducing repetition. Postulate:

* any repeated operation/string/number should be put into it's own private, auxiliary constant, e.g. `nonIPKeysWidth = DPortWidth + ProtoWidth`, `sipDipIPv4Width = 2*IPv4Width`, etc.
* public constants should be composed from those auxiliary constants
```
KeyWidthIPv4 = sipDipIPv4Width + nonIPKeysWidth
```

Since this happens all at compile time: no performance hit, but more readable explanation on how these constants are composed.

You can think about it.

Reply:
TBH I'm not 100% sure that an overabundance of consts for the sake of maximum separation necessarily adds to readability, but as you said it's a matter of opinion. Since it's not that many here I've slightly reworked it to reflect that and, more importantly, have hidden all consts that are not needed outside of this package (because I _do_ think that's good practice)...